addons/stock_landed_costs/models/product.py

```diff
@@ -11,6 +11,7 @@
     ('by_volume', 'By Volume'),
 ]
 
+
```

Comment:
This commit does not seem to contain anything interesting. Please discard it.

Except the removal of an import. Why ? Is it a fix or a mistake ?